layout/src/style/projects.scss

```
@@ -55,13 +55,25 @@
             padding: .5rem 1rem;
 
             &::before {
-              background-color: #dea584;
+              background-color: #ccc;
               border-radius: 100px;
               content: "";
               height: .9rem;
               margin-right: .5rem;
               width: .9rem;
             }
+
+            &.rust::before {
+              background-color: #dea584;
+            }
+
+            &.css::before {
+              background-color: #563d7c;
+            }
+
+            &.html::before {
+              background-color: #fc7f02;
+            }
```

As we are using Sass we could just nest the class inside the `::before` to improve readability and also take advantage of Sass nesting capabilities!

This is not tested, but Im pretty sure you can accomplish something similar!

```suggestion
&::before {
  background-color: #ccc;
  border-radius: 100px;
  content: "";
  height: .9rem;
  margin-right: .5rem;
  width: .9rem;

  &.rust {
    background-color: #dea584;
  }

  &.css {
    background-color: #563d7c;
  }

  &.html {
    background-color: #fc7f02;
  }
}
```